.gitignore

```diff
@@ -81,7 +81,7 @@ dojo/uploads/threat/*
 *.sqlite
 *.db
 celerybeat.pid
-*.env*
+./*.env*
```

Comment:
I use this `dojo/settings/.env.prod` in my local environment, which now becomes unignored. Not sure if this is bad or can be avoided. I think we should try to avoid accidentally commiting "prod credentials" into git.

Reply:
I changed it back to how it was and added an exception rule for the docker environment files